packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -494,7 +494,7 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
 }

```

Comment:
It doesn't look like anything changed in this file. Any reason to commit it?

Reply:
this is auto generated.